### DISCUSSION TITLE
Could not find GNU_MAKE_PROGRAM using the following names: gmake

### DISCUSSION THREAD

**Discussion by HSQ8** [florisboard/florisboard#2217] [Q&A]
On the most recent master branch, I am running into this issue when building inside android studio in windows 11

```
-- Configuring incomplete, errors occurred!
See also "E:/florisboard/app/.cxx/Debug/4d3u652l/arm64-v8a/CMakeFiles/CMakeOutput.log".
...
CMake Error at nlp/icu4c/CMakeLists.txt:7 (find_program):
  Could not find GNU_MAKE_PROGRAM using the following names: gmake

```

I checked that cmake was indeed included in the SDK tools for this project, and afaik cmake is analogous to gmake? except gmake is on GNU. Do you recommend I just switch over to a linux environment completely or did I miss an easy fix for this problem? Thanks!

**Comment by patrickgold** [comment]
I mean switching over to a Linux env will definitely help resolving your issue, as that's what the build scripts were primarly built for, however I also try to fix and improve the build script so it compiles on other platforms as well. As for dependencies, GNU gmake is definitely required though to get the ICU4C dependency to compile.

There are more people having build issues with the new build config, so I am currently trying to gather some information at #2218, please make sure to check out that issue as I will also post there once I have some fixes to try out.
